.travis.yml

```diff
@@ -11,15 +11,15 @@ sudo: required
 matrix:
   fast_finish: true
   include:
-    - php: 7.2
+    - php: 7.3
       env:
         - PRODUCT_NAME=suite
 
-    - php: 7.2
+    - php: 7.3
```

Comment:
For the future: It would probably help to have one job running php74 (max) just to spot any issues with that version.
For now we can ignore.